plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.tsx

```diff
@@ -8,19 +8,21 @@ import { Card } from '@wordpress/components';
  * Internal dependencies
  */
 import './product-card.scss';
-import { Product } from '../product-list/types';
+import { Product, ProductType } from '../product-list/types';
 import { appendUTMParams } from '../../utils/functions';
 
 export interface ProductCardProps {
-	type?: string;
+	type?: ProductType;
 	product: Product;
 }
 
 function ProductCard( props: ProductCardProps ): JSX.Element {
-	const { product } = props;
+	const { product, type = ProductType.classic } = props;
```

Comment:
This feels nitpicky, but I wonder if the default should be `ProductType.extension` since the alternative is `ProductType.theme`. Not a blocker and welcome to be ignored.

Reply:
I realised in a later comment I might be wrong here. Please see my other comment further down and ignore this one if it's unnecessary.

Reply:
Classic was the name we had in code already and I didn't want to remove it in this PR just to keep things clean.
Could we address it in another PR?

Reply:
I removed classic now and replaced it with extension.
Originally I didn't know what it [was used for here](https://github.com/woocommerce/woocommerce/pull/40195/files#diff-d33ce5ad0ffbbd53cd5beb965bbe34efd3f3dbf41a12e18c258dbd04eb02e6c2L17) so I kept it, but it's not used for anything.